ui/jobs/bar.d.ts

```diff
@@ -1,8 +1,7 @@
-// TODO: remove this when ui/jobs/jobs.js is converted to typescript
```

Comment:
Why this comment is removed

Reply:
Hmmm, weird. I didn't delete it, so it must have, but it dealt with other comments just fine? I'll add it back, but good catch.